packages/lesswrong/lib/reactRouterWrapper.tsx

```diff
@@ -29,7 +29,13 @@ const isLinkValid = (props: LinkProps): props is HashLinkProps => {
 };
 
 export const Link = ({eventProps, ...props}: LinkProps) => {
-  const { captureEvent } = useTracking({eventType: "linkClicked", eventProps: {to: props.to, ...(eventProps ?? {})}})
+  const { captureEvent } = useTracking({
+    eventType: "linkClicked",
+    eventProps: {
+      to: String(props.to),
```

Comment:
Event props should be `JSON.stringify`able, but `props.to` can be a function. This conversion is still a little janky, but at least you can _see_ it's janky instead of just letting it be hidden.